Trappist/src/Promact.Trappist.Web/wwwroot/app/app.module.ts

```diff
@@ -1,11 +1,13 @@
 ﻿import { NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
-import { RouterModule } from '@angular/router';
+import { RouterModule } from "@angular/router";
```

Comment:
do you think it should be single quote instead of double quote? did you check for tslint issues?